src/Microsoft.TestPlatform.CommunicationUtilities/TestRequestSender.cs

```diff
@@ -771,7 +771,9 @@ private static ICommunicationEndPoint SetCommunicationEndPoint(TestHostConnectio
 
 internal class MessageConverter
 {
+#pragma warning disable IDE0060 // Remove unused parameter // TODO: Use or remove this parameter and the associated method
```

Comment:
@nohwnd Is it ok to remove or do you plan to use these later?

Reply:
This was my "template" for compatibility work. Leave it there for a bit please.